### DISCUSSION TITLE
List of country codes (in M3U/JSON format)

### DISCUSSION THREAD

**Discussion by tpkahlon** [iptv-org/community#201] [Ideas]
I revisited Jackal [repository](https://github.com/tpkahlon/jackal) after a while and noticed that `index.m3u` link no longer exists.

This link served as a backbone for `jackal.surge.sh` project.

In order to get latest revisions made in channels folder over to Jackal, It would be great if you can create a M3U file that holds all country codes like previous one or create an endpoint where I can get an array of all the available channel codes.

I hope jackal.surge.sh can stay fluid with ongoing changes made into IPTV repository.

**Comment by BellezaEmporium** [comment]
I'll tag @freearhey so that he can talk about it.

**Reply by tpkahlon** [reply]
@freearhey @LaneSh4d0w 

No need for additional work on this. I have managed to make it work on my end. Jackal should be fluid with new changes and additions in the repository going forward.
